### PR TITLE
Follow-up to PR3683 Minor formatting edits

### PR DESCRIPTION
**Pull request by bmcelvee**
Continuation of https://github.com/openshift/openshift-docs/pull/3683.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@adellape @ahardin-rh Please peer review. Thanks! 😸

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -649,17 +649,25 @@ $ oadm diagnostics MetricsApiProxy
...
 
-The link:https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift Agent] can be used to gather application metrics from pods running within the {product-title} cluster. These metrics can then be viewed from the console or accessed via the Hawkular Metrics REST API.
+The link:https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift
+Agent] can be used to gather application metrics from pods running within the
+{product-title} cluster. These metrics can then be viewed from the console or
+accessed via the Hawkular Metrics REST API.
 
-In order for these metrics to be gathered, your pod needs to expose a Prometheus or Jolokia endpoint and create a special configmap which defines where the metric endpoint is located and how the metrics should be gathered. More information can be found under the link:https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift Agent documentation].
+In order for these metrics to be gathered, your pod needs to expose a Prometheus
+or Jolokia endpoint and create a special configmap which defines where the
```
Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: `ConfigMap`

Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: comma before which

**Comment by ahardin-rh** [comment]
Just 2 comments from me. Otherwise, looks good!

**Comment by bmcelvee** [comment]
Thanks, @ahardin-rh! Changes committed.

**Comment by bmcelvee** [comment]
Revision history in https://github.com/openshift/openshift-docs/pull/3683.

**Comment by ahardin-rh** [comment]
added a milestone label to this one :balloon:

**Comment by ahardin-rh** [comment]
@bmcelvee now that this is merged, you can remove the to-followup label from the original PR

**Pull request merged**